components/contribution-flow/ContributionSummary.js

```diff
@@ -48,11 +48,12 @@ const Amount = styled(Span)`
   text-align: right;
 `;
 
-const ContributionSummary = ({ collective, stepDetails, stepSummary, stepPayment, currency }) => {
+const ContributionSummary = ({ collective, stepDetails, stepSummary, stepPayment, currency, isCrypto }) => {
   const intl = useIntl();
   const totalAmount = getTotalAmount(stepDetails, stepSummary);
   const pmFeeInfo = getPaymentMethodFees(stepPayment?.paymentMethod, totalAmount, currency);
   const platformContribution = stepDetails.platformContribution || 0;
+  const currencySymbol = isCrypto ? stepDetails.currency.value : currency;
```

Comment:
Same as before, why not rely simply on `currency`?

Reply:
Corrected as per above; https://github.com/opencollective/opencollective-frontend/pull/6689#discussion_r684481213